astro/src/content/blog/announcing-fusionauth-1-50.mdx

```diff
@@ -0,0 +1,122 @@
+
+The ability to create custom scopes and consent screens allows you to safely and securely extend user data access to partners and third party developers.
+
+This release also brings FusionAuth into better compliance with a number of specifications such as OIDC. but don't worry, we've also built in backwards compatibility. The [Scopes documentation](/docs/lifecycle/authenticate-users/oauth/scopes) discusses how you might want to update your configuration.
+
+Prompting the user for consent is limited to certain paid plans. Learn more by reviewing [the pricing page](/pricing) or [contacting our sales team](/contact).
+
+## UserInfo Lambda
+
+The [UserInfo endpoint](/docs/lifecycle/authenticate-users/oauth/endpoints#userinfo), as defined in the OIDC specification, takes an access token and returns information about the user. Per the specification:
+
+> The UserInfo Endpoint is an OAuth 2.0 Protected Resource that returns Claims about the authenticated End-User.
+
+FusionAuth has supported this endpoint since inception, but this release now allows you to customize the response. You use a lambda, which is a snippet of JavaScript assigned to an application.
+
+Customizing the UserInfo response allows you a lot of flexibility. You can:
+
+* Remove any personally identifiable information (PII) that is not required.
+* Add additional claims needed by callers of the endpoint, even if they depend on the access token.
+* Augment the provided claims by calling other APIs using [Lambda HTTP Connect](/docs/extend/code/lambdas/#using-lambda-http-connect).
+
+Here's an example lambda which removes PII, but leaves the email domain available for any service consuming it. The `sub` claim is also untouched.
+
+```
+function populate(response, user, registration, jwt) {
+  response.email = user.email.replace(/[^@]*@/,"deleted@");
+  response.given_name = "deleted";
+  response.family_name = "deleted";
+  response.birthdate = null;
+}
+```
+
+You can [manage](/docs/extend/code/lambdas/#managing-lambdas) and [test](/docs/extend/code/lambdas/testing) the UserInfo lambda just as you do any other lambda.
+
+## Improved Setup Wizard
+
+The initial experience of getting going with FusionAuth can be a bit confusing, [quickstarts notwithstanding](/docs/quickstarts/). There are a lot of "nerd knobs" to tweak, as [Matt Hathcock, one of our Sales Engineers](/blog/sales-engineering), puts it.
+
+This level of customization is powerful when you are familiar with it, but it can be frustrating when first starting. In this release, there's a screen which walks new admin users through FusionAuth setup. It covers:
+
+* Creating an application, including setting up the redirect URL.
+* Setting up your SMTP configuration, which is important for workflows like "forgot password".
+* Creating an API key, which is great for automation.
+* Entering a license key if the user has a paid plan.
+
+TODO image
```

Comment:
Still need image.

Reply:
Yup, I'm waiting until release to capture the image.